index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>SD Global Conference</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous" />
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+    type="text/css" />
+</head>
+
+<body>
+  <div class="top-bar-section">
+    <div class="top-bar-container">
+      <ul class="top-bar">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
```

Comment:
```suggestion
          <a>English</a>
        </li>
        <li>
          <a>My Page</a>
        </li>
        <li>
          <a>Logout</a>
```
For semantic HTML, perhaps it would be preferred that these be links instead of headings. This helps with SEO when we use semantic tags.
To increase their size, you can always add a CSS class and style them.